### PR TITLE
Azure Sentinel description updated

### PR DESCRIPTION
**Pull request by rshunim**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/XSUP-21188)

## Description
Azure Sentinel description in the Marketplace was updated.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.yml [hunk 1]
```diff
@@ -126,7 +126,7 @@ configuration:
   type: 8
   section: Connect
   advanced: true
-description: Use the Azure Sentinel integration to get and manage incidents and get related entity information for incidents.
+description: "Microsoft Sentinel is a scalable, cloud-native solution that provides: Security information and event management (SIEM) Security orchestration, automation, and response (SOAR)."
```
Comment (dorschw) [inline Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.yml:129]:
```suggestion
description: Microsoft Sentinel is a scalable, cloud-native solution that provides: Security information and event management (SIEM) Security orchestration, automation, and response (SOAR).
```

**Review by dorschw — APPROVED** [review]
Nice

File: Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.yml [hunk 2]
```diff
@@ -126,7 +126,7 @@ configuration:
   type: 8
   section: Connect
   advanced: true
-description: Use the Azure Sentinel integration to get and manage incidents and get related entity information for incidents.
+description: Microsoft Sentinel is a scalable, cloud-native solution that provides: Security information and event management (SIEM) Security orchestration, automation, and response (SOAR).
```
Comment (dorschw) [inline Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.yml:129]:
```suggestion
description: "Microsoft Sentinel is a scalable, cloud-native solution that provides: Security information and event management (SIEM) Security orchestration, automation, and response (SOAR)."
```

**Comment by ShirleyDenkberg** [comment]
@dorschw Doc review completed.

**Pull request merged**